bus/bus.go

```diff
@@ -1178,17 +1178,38 @@ func (b *bus) handleGETAlerts(c jape.Context) {
 	c.Encode(b.alerts.Active())
 }
 
-func (b *bus) handlePOSTAlertsDismiss(c jape.Context) {
+func (b *bus) handlePOSTAlertsDismiss(jc jape.Context) {
 	var ids []types.Hash256
-	if c.Decode(&ids) != nil {
+	if jc.Decode(&ids) != nil {
 		return
 	} else if len(ids) == 0 {
-		c.Error(errors.New("no alerts to dismiss"), http.StatusBadRequest)
+		jc.Error(errors.New("no alerts to dismiss"), http.StatusBadRequest)
 		return
 	}
 	b.alerts.Dismiss(ids...)
 }
 
+func (b *bus) handlePOSTAlertsRegister(jc jape.Context) {
+	var alert alerts.Alert
+	if jc.Decode(&alert) != nil {
+		return
+	}
+	if alert.ID == (types.Hash256{}) {
```

Comment:
`nit` probably good to have this in a `validate` method on the alert itself - keeps the handler short and it's useful to have